themes/bootstrap3/less/bootstrap.less

```diff
@@ -1,19 +1,20 @@
 /* #LESS> */
 @import "vendor/bootstrap/bootstrap";
 @import "vendor/font-awesome/font-awesome";
+@import "vendor/autocomplete";
 /* <#LESS */
 /* #SCSS>
 $fa-font-path: "../../bootstrap3/css/fonts";
 @import "vendor/bootstrap";
 @import "vendor/font-awesome/font-awesome";
+@import "vendor/autocomplete";
```

Comment:
@crhallberg, since the font-awesome and autocomplete imports appear identical for LESS and SCSS, is there any reason to keep them inside these conditional blocks? It looks like the conditional part could end after the bootstrap import, since that works differently in each case.